js/dropdown/shared/Dropdown.svelte

```diff
@@ -250,9 +250,12 @@
 
 <style>
 	.icon-wrap {
+		position: absolute;
+		top: var(--size-2);
```

Comment:
Nice! My concern is that fixing the top position to be --size-2 will not render perfectly in themes that have a different font size. For example, this theme is set text-md to be 18px and the arrow is not vertically centered.

<img width="305" alt="Screenshot 2024-04-05 at 12 58 16 PM" src="https://github.com/gradio-app/gradio/assets/7870876/d185e430-a6b7-4c4d-9b39-46b19b4c5f66">

Would ideally be properly centered vertically

Reply:
https://chat.openai.com/share/d5a7ada5-93e5-4119-86ae-c2d3bec553b4